BearlyHumans/squirrelville
# Horizon Fog & Clouds

to test:

- load into island scene 
- look out into the water
- make sure the clouds and fog look okay
- use dev door to move higher up the island
- lookout on the water, it should be blended with the sky so there is no harsh line
